code/Controllers/ContentController.php

```diff
@@ -415,9 +386,7 @@ public function SiteConfig()
 
     /**
      * Returns an RFC1766 compliant locale string, e.g. 'fr-CA'.
-     * Inspects the associated {@link dataRecord} for a {@link SiteTree->Locale} value if present,
-     * and falls back to {@link Translatable::get_current_locale()} or {@link i18n::default_locale()},
-     * depending if Translatable is enabled.
+     * Inspects the associated {@link dataRecord} for a {@link SiteTree->Locale} value if present, and falls back to {@link i18n::default_locale()}
```

Comment:
You've changed this to no longer look for a `SiteTree->Locale` value. We should either look for that value (if anything in core would add it) or remove the reference to that in the phpdoc

Reply:
Have removed the phpdoc